ios/FluentUI.Demo/FluentUI.Demo/Demos/PersonaButtonCarouselDemoController.swift

```diff
@@ -268,4 +279,16 @@ class PersonaButtonCarouselDemoController: DemoTableViewController {
         PersonaData(email: "carlos.slattery@contoso.com", subtitle: "Software Engineer")
     ]
 
+    private let groupPersonas: [PersonaData] = [
```

Comment:
Can you not re-use the other persona data? I'm not sure I love the structure of this test data.

Reply:
It's possible, but I created this data to resemble the specs design shared with me. I've shared the files with you on teams.